force-app/main/default/lwc/gsChecklistItem/gsChecklistItem.js

```diff
@@ -29,6 +38,13 @@ export default class GsChecklistItem extends NavigationMixin(LightningElement) {
                 if (data) {
                     this.namespace = data;
                 }
+                this.linkLabel = ( this.item.link.label + ' ' + this.labels.opensInNewLink);
+                if( !(this.item.primaryBtn) === undefined){
```

Comment:
Actually, after testing, this works as intended. It only sets the information for variables primaryBtnLabel and secondaryBtnLabel when what comes through this.item.primaryBtn or this.item.secondaryBtn is not undefined.

Reply:
I'm not able to reproduce a situation where this expression can evaluate to be `true`. `!(anyValue)` will always be either `true` or `false`, and when compared with undefined it will always be `false`.

Reply:
I understand, @bdvorachek. However, in this case, some of the items passed down from the parent component have an item.primaryBtn (or item.secondaryBtn) while others do not. In the cases which nothing is passed down from item.primaryBtn (or item.secondaryBtn), the attribute actually came as undefined. This was the only way in which I was able of avoiding "Cannot read 'label' from 'undefined'" for items that had no info for primaryBrn or secondaryBtn while actually creating the aria-label for the items that did. If you create a scratch org and remove this validation, you will be able of seeing this error. And adding back the validation, it will be solved. Let me know if you wanna meet regarding this.

Reply:
@bdvorachek please dismiss my last comment. Using hasPrimaryBtn and hasSecondaryBtn methods is indeed a much better solution. I have modified and pushed this changes. Thank you!!